src/hrtf/reader.c

```diff
@@ -121,6 +119,100 @@ static int getArray(struct MYSOFA_ARRAY *array, struct DATAOBJECT *dataobject) {
 	return MYSOFA_OK;
 }
 
+static int isNonStandardVariable(struct DIR *dir) {
+	if (!strcmp(dir->dataobject.name, "ListenerPosition")) {
```

Comment:
you do not need to check for the long variable names. this is already done in the function above.

Reply:
In this context, true. But a generic function that checks if variable is standard or not should probably check all of them, otherwise the name might be confusing.

Reply:
actually, you do not need this function

Reply:
I removed it.